integration-tests/modules/__tests__/customer-group/admin/list-customer-group-customers.ts

```diff
@@ -54,7 +54,7 @@ medusaIntegrationTestRunner({
         )
 
         const response = await api.get(
-          `/admin/customer-groups/${group.id}/customers`,
```

Comment:
This is an anti-pattern and we should just use the `customers` endpoint with a filter (as done for eg. for products and price lists)